README.md

```diff
@@ -23,28 +23,41 @@ Check the [wiki][] for more tips
 Installation
 ------------
 
-1. Using Package Control, install "FileDiffs"
+### Package Control (Easiest)
 
-Or:
+1. Using Package Control, install `FileDiffs`
+
+### Sublime Text 3
 
 1. Open the Sublime Text Packages folder
-    - OS X: ~/Library/Application Support/Sublime Text 3/Packages/
-    - Windows: %APPDATA%/Sublime Text 3/Packages/
-    - Linux: ~/.Sublime Text 3/Packages/ or ~/.config/sublime-text-3/Packages
+    - OS X: `~/Library/Application Support/Sublime Text 3/Packages/`
+    - Windows: `%APPDATA%/Sublime Text 3/Packages/`
+    - Linux: `~/.Sublime Text 3/Packages/` or `~/.config/sublime-text-3/Packages`
+
+2. Clone this repo
+
+    ```
+    # Over SSH
+    git clone git@github.com:colinta/SublimeFileDiffs
+
+    # Over HTTPS
+    git clone https://github.com/colinta/SublimeFileDiffs.git
+    ```
 
-2. clone this repo
 3. Install keymaps for the commands (see Example.sublime-keymap for my preferred keys)
 
 ### Sublime Text 2
 
 1. Open the Sublime Text 2 Packages folder
-2. clone this repo, but use the `st2` branch
-
-       git clone -b st2 git@github.com:colinta/SublimeFileDiffs
+2. Clone this repo, but use the `st2` branch
```

Comment:
I wonder if we should remove ST2 support altogether - I don't maintain that branch at all... on the other hand, I guess it doesn't hurt to have it. 🤔

Reply:
It wouldn't hurt to leave it. Later, it could be removed or edited with a disclaimer that it is not actively maintained.